amgut/lib/data_access/sql_connection.py

```diff
@@ -53,6 +53,7 @@ def __init__(self):
         self._connection = None
         self._post_commit_funcs = []
         self._post_rollback_funcs = []
+        self.auto_rollback = False
```

Comment:
The problem is `deleteAGParticipantSurvey` makes a lot of changes to the database, and since we are committed to only using access functions and not direct SQL in the tests, we can't set up a tearDown that won't error on every other test. It makes more sense to just rollback after the test, which also ensures 100% that the DB goes back to before the tests.